react-validated-input-component/src/validated-input.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+
+export default function ValidatedInput() {
+  const [isPassword, setIsPassword] = useState('');
+  const [tooShort, setTooShort] = useState(false);
+
+  function isEight(e) {
+    setIsPassword(e.target.value);
+    if (e.target.value === '') {
+      setTooShort(false);
```

Comment:
According to the spec, if there is no password you should display the 'too short' message.